src/main/webapp/pages/UserPage.html

```diff
@@ -47,8 +47,8 @@ <h1 class="center">Create Recipe</h1>
             <!-- <option value="public">public</option> -->
             </select>
             <br><br>
-            <label for="image">Upload an image</label>
-            <input type="file"  accept="image/*" name="image" id="image">
+            <!-- <label for="image">Upload an image</label>
```

Comment:
Are you leaving this to keep working on it later?

Reply:
yeah I tried to make it work but it's really not lol so for the MVP I'll leave it and work on it again later

Reply:
Ok, then I would say add a comment with a to-do, and maybe the reason it's not MVP.